### PR TITLE
Remove email from license submit

### LINKED ISSUE #411: Errors submitting license when a close diff is found
The following response was sent when submitting a new license with [this text](https://raw.githubusercontent.com/viennacl/viennacl-dev/master/LICENSE):

`Unexpected error while posting to github, please e…turned 401 status code - message Bad credentials\n`

### PR DESCRIPTION
**Pull request by goneall**
Fixes #411

Also fixes a broken link to license matching guidelines.

Signed-off-by: Gary O'Neall <gary@sourceauditor.com>

### CODE REVIEW

**Comment by goneall** [comment]
This PR fixes issue #411 but does not fix #412.

I could not find the cause of #412, but it was likely caused by the recent changes to submit license.

@rtgdk - Could you take a look and see if there is something obvious I missed?

**Comment by goneall** [comment]
I fixed the OK and link buttons - this should now be good to go.

**Review by rtgdk — APPROVED** [review]
Looked at the codebase for other usages of `userEmail`. Found it only on the src/api level of in LicenseNamespace views which is deprecated for now. The one distinct use was in `forms.py`
```
class LicenseRequestForm(forms.Form):

    def __init__(self, *args, **kwargs):
        if 'email' in kwargs:
            self.email = kwargs.pop('email')
        else:
            self.email = ""
        super(LicenseRequestForm, self).__init__(*args,**kwargs)
        self.fields["userEmail"] = forms.EmailField(label='Email', initial=self.email)
        ```
        which I think we can ignore since if user is logged only then email will be populated else empty, but we won't be asking it on the UI nor using it when submitting the license.
        
        Changes LGTM

File: src/app/views.py [hunk 1]
```diff
@@ -1171,7 +1171,7 @@ def issue(request):
                         listVersionAdded, licenseSourceUrls, licenseHeader, licenseNotes, licenseText)
                     now = datetime.datetime.now()
                     licenseRequest = LicenseRequest(licenseAuthorName=licenseAuthorName, fullname=licenseName, shortIdentifier=licenseIdentifier,
-                        submissionDatetime=now, userEmail=userEmail, notes=licenseNotes, xml=xml)
+                        submissionDatetime=now, notes=licenseNotes, xml=xml)
```
[the remark this answers is not in the document]
Comment (rtgdk) [reply src/app/views.py:1174]: This will be somewhat large change

[the remark this answers is not in the document]
Comment (goneall) [reply src/app/views.py:1174]: I created issue #421 to track

**Pull request merged**